pkg/cloudprovider/aws/launchtemplate.go

```diff
@@ -192,26 +193,51 @@ func (p *LaunchTemplateProvider) getSecurityGroupIds(ctx context.Context, provis
 
 func (p *LaunchTemplateProvider) getUserData(ctx context.Context, provisioner *v1alpha3.Provisioner, constraints *Constraints) (string, error) {
 	var userData bytes.Buffer
-	userData.WriteString(fmt.Sprintf("[settings.kubernetes]\napi-server = \"%s\"\n", provisioner.Spec.Cluster.Endpoint))
-	userData.WriteString(fmt.Sprintf("cluster-name = \"%s\"\n", *provisioner.Spec.Cluster.Name))
+	userData.WriteString(fmt.Sprintf(`#!/bin/bash
+yum install -y https://s3.amazonaws.com/ec2-downloads-windows/SSMAgent/latest/linux_amd64/amazon-ssm-agent.rpm
```

Comment:
`ec2-downloads-windows`? Weird, I tested this and it seems to work, just odd bucket name I guess

Reply:
Yes, I believe it is due to the fact that the ssm-agent was originally developed for Windows, and later was extended to support Linux as well.